app/controllers/content_items_controller.rb

```diff
@@ -101,6 +101,10 @@ def with_locale
     I18n.with_locale(@content_item.locale || I18n.default_locale) { yield }
   end
 
+  def setup_content_navigation_ab_testing
```

Comment:
Is this method needed? I think it was removed in a previous commit.

Reply:
Good spot, sorry, not one of ours so it's got to be a merge error. We've just pushed a commit removing it.